src/shared/ScreenLayout.js

```diff
@@ -65,7 +66,7 @@ const ScreenLayout = (props) => {
       </BodyContainer>
 
       <Footer>
-        {btnLabel ? (
+        {btnLabel && selectedDialectId || selectedGrammarId ? (
```

Comment:
Would it work if you did `btnLabel || selectedGrammarId` ?

Reply:
No, and selectedGrammarId is needed. 🤔